Makefile

```diff
@@ -178,11 +180,17 @@ postgres-clean-%:
 postgres-check-%: postgres-%
 	$(MAKE) -C $(POSTGRES_INSTALL_DIR)/build/$* MAKELEVEL=0 check
 
+.PHONY: libcommunicator
+libcommunicator:
```

Comment:
Should this depend on `cargo-target-dir`? Not sure if we need it for anything, just noticed it's there.